### ISSUE TITLE
How did you decide epochs ?

### ISSUE THREAD

**Issue by ken-take-it-so-so** [yantijin/ScoreGradPred#1]
Hello.  
I read "ScoreGrad: Multivariate Probabilistic Time Series Forecasting with Continuous Energy-based Generative Models".  
How did you decide epochs?
I would appreciate your cooperation.

**Comment by yantijin** [comment]
We use grid-search method for hyper-parameter settings.
It's a good question, as we noted in the paper and appendix, the value of the train loss is not directly correlated with the performance.
